HW04/ekaterina_balabanovich/lib/work_state/work_state_rejected.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+class WorkStateRejected < WorkState
+  def process(members, homework)
+    WorksManager.instance.notify(members: members, task: homework, status: :reject)
+  end
+end
```

Comment:
We usually do something like this if we want to represent such structure 
```suggestion
module WorkStates
  class Rejected < Base
    def process(members, homework)
      WorksManager.instance.notify(members: members, task: homework, status: :reject)
    end
  end
end

WorkState::Rejected.new
```